docs/az/docs/index.md

```diff
@@ -0,0 +1,469 @@
+
+{!../../../docs/missing-translation.md!}
+
+
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI framework, yüksək performans, sadə öyrənmək, çevik kodlamaq, istehsal üçün hazır</em>
+</p>
+<p align="center">
+<a href="https://github.com/tiangolo/fastapi/actions?query=workflow%3ATest" target="_blank">
+    <img src="https://github.com/tiangolo/fastapi/workflows/Test/badge.svg" alt="Test">
+</a>
+<a href="https://codecov.io/gh/tiangolo/fastapi" target="_blank">
+    <img src="https://img.shields.io/codecov/c/github/tiangolo/fastapi?color=%2334D058" alt="Coverage">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/v/fastapi?color=%2334D058&label=pypi%20package" alt="Package version">
+</a>
+</p>
+
+---
+
+**Sənədlər**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**Mənbə kodu**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI müasir, sürətli (yüksək performanslı), standart Python tipli göstərişlərə əsaslanaraq Python 3.6+ ilə API qurmaq üçün veb framework-ü.
+
+Aşağıdakılar əsas xüsusiyyətlərdir:
+
+* **Sürətli**: Çox yüksək performans, **NodeJS** və **Go** ilə bərabər (Starlette və Pydantic-ə təşəkkürlər). [Ən sürətli Python frameworklərindən biri](#performans).
+
+* **Çevik kodlama**: Xüsusiyyətləri inkişaf etdirmə sürətini təxminən 200-300% artırın. *
+* **Daha az boşluq**: İnsan (developer) tərəfindən törədilən səhvlərin təxminən 40% -ni azaldın. *
+* **İntuitiv**: Böyük redaktor dəstəyi. <abbr title="auto-complete, autocompletion, IntelliSense olaraq da bilinir">Tamamlama</abbr> həryerdə. Xəta müəyyənləşdirmək üçün daha az vaxt.
+* **Asan**: İstifadəsi və öyrənməsi asan olması üçün hazırlanmışdır. Sənədləri oxumaq üçün daha az vaxt.
+* **Qısa**: Kodun təkrarlanmasını minimuma endirin. Hər bir parametr bəyannaməsindən birdən çox xüsusiyyət. Daha az səhv.
+* **Sağlam**: İstehsal üçün hazır kodu əldə edin. Avtomatik interaktiv sənədlərlə.
```

Comment:
"Sağlam" also means healthy, which is true for robust work, however, in this context, "Qüvvətli" or something similar that means "Strong" will be a better use. Because it explains being ready for production load.

Reply:
"kodu" -> "kod"
Reason: you still have to work on the code, so, you are getting result at the end. Using "kodu" will mean that you are getting final product now.